### DISCUSSION TITLE
v2.7.4 / [预览] v2.8.2

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4320] [分享 / Share]
- 修复 v4.2 播放器下视频相关功能失效. (#4303)

注: 下载字幕还是会失败, 属于另一个问题 (#4319), 需要另外再修复.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.7.4'>v2.7.4 / [预览] v2.8.2</a>.</em>

**Comment by kethoven** [comment]
厉害，终于好了，还以为需要等很长时间呢
